services/app-api/src/postgres/contractAndRates/findContractWithHistory.test.ts

```diff
@@ -124,7 +124,9 @@ describe('findContract', () => {
         if (threeContract instanceof Error) {
             throw threeContract
         }
-        expect(threeContract.revisions).toHaveLength(4)
```

Comment:
I'm very hesitant to lose the specific values in this test. This test was written to check that we get the expected number of revisions everywhere. I'd rather mark this test as skip or something than adapt it to work with the interim solution.

Reply:
Let's mark as skip.